dockerfiles/Dockerfile

```diff
@@ -132,6 +132,7 @@ RUN echo "deb http://deb.debian.org/debian bookworm-backports main" >> /etc/apt/
     libpsl5 \
     libbrotli1 \
     libsasl2-2 \
+    libsasl2-modules-gssapi-mit \
```

Comment:
I presume this requires no additional build time dependencies?

Reply:
No, builds work fine without anything additional and with this flb finds the library